### PR TITLE
Cleanup flake8 warnings and formatting.

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: lib/ec2imgutils/ec2uploadimg.py [hunk 1]
```diff
@@ -154,7 +157,7 @@ def _change_mount_point_permissions(self, target, permissions):
         command = 'chmod %s %s' % (permissions, target)
         result = self._execute_ssh_command(command)
 
-        return 1
+        return result
```
[the remark this answers is not in the document]
Comment (rjschwei) [reply lib/ec2imgutils/ec2uploadimg.py]: In the current code the return value is ignored, But yes we might as well return the result.

File: lib/ec2imgutils/ec2uploadimg.py [hunk 2]
```diff
@@ -390,7 +393,7 @@ def _create_storage_filesystem(self, device_id):
         command = 'mkfs -t ext3 %s' % filesystem_partition
         result = self._execute_ssh_command(command)
 
-        return 1
+        return result
```
[the remark this answers is not in the document]
Comment (rjschwei) [reply lib/ec2imgutils/ec2uploadimg.py]: Ditto

File: lib/ec2imgutils/ec2uploadimg.py [hunk 3]
```diff
@@ -521,7 +524,7 @@ def _dump_root_fs(self, image_dir, raw_image_name, target_root_device):
                                                 target_root_device)
         result = self._execute_ssh_command(command)
 
-        return 1
+        return result
```
[the remark this answers is not in the document]
Comment (rjschwei) [reply lib/ec2imgutils/ec2uploadimg.py]: Ditto

**Comment by rjschwei** [comment]
Please bump the version by 0.0.1 
specfile and lib/ec2imgutils/VERSION

the cleanup caught a few latent bugs. Thanks

**Comment by smarlowucf** [comment]
Okay, will also rebase against master and integrate the formatting check into travis.

**Comment by smarlowucf** [comment]
Bumped version, updated travis config with flake8 check, and cleaned up remaining warnings in the scripts.

**Pull request merged**